requirements-dev.txt

```diff
@@ -12,6 +12,8 @@ sphinx-gallery
 pydata_sphinx_theme
 
 # ATen lib
+typing_extensions
```

Comment:
Could you re-sort them in alphabetical order?

Reply:
Done